department-of-veterans-affairs/vets-website
# Feature/18800 remove conditional cta for cheetah mvp

## Description
Age is no longer a criteria for  project Cheetah. Remove the conditional CTA 
 
### How to test

1. Turn ON va_online_scheduling_cheetah feature flag; turn OFF va_online_scheduling_homepage_refresh
2. Go to the appointments page to see the text change (1st screenshot)
3. Turn ON va_online_scheduling_cheetah feature flag; turn ON va_online_scheduling_homepage_refresh 
4. Go to the appointments page to see the text change (2dn screenshot)

## Testing done
local and unit test

## Screenshots
### When homepage refresh flag is OFF
![image](https://user-images.githubusercontent.com/54327023/107235029-44ab2100-69f2-11eb-93ae-162ceb34911a.png)


### When homepage refresh flag is ON
![image](https://user-images.githubusercontent.com/54327023/107235222-73c19280-69f2-11eb-9b5b-5ea0e9c457ed.png)


## Acceptance criteria
- [ ] The Cheetah CTA should read 'You may be eligible to receive the COVID-19 vaccine at a VA Location`
- [ ] Textual update present whenever va_online_scheduling_homepage_refresh is turn on or off


## Definition of done
